oshi-core/src/main/java/oshi/hardware/CentralProcessor.java

```diff
@@ -257,13 +257,21 @@
      * @return Number of seconds since boot.
      */
     long getSystemUptime();
+    
+    /**
+     * Get the System boot time.
+     * 
+     * @return Number of seconds since system last booted.
+     */
+    long getBootTime();
 
     /**
      * Get the number of logical CPUs available for processing. This value may
      * be higher than physical CPUs if hyperthreading is enabled.
      *
      * @return The number of logical CPUs available.
      */
+
```

Comment:
(Extremely minor point) Lots of extra whitespace lines here.  Do you use a formatter in your IDE?  For the project we use the "Sun Java Conventions" except we do spaces instead of tabs.